### PR TITLE
Update Kaggle-Orpheus_(3B)-TTS.ipynb to fix issue #24

### PR DESCRIPTION
**Pull request by rupaut98**
The training script failed with a ValueError because the default data collator couldn't handle the variable sequence lengths inherent in the dataset (due to different text/audio lengths). This PR fixes the issue #24 by explicitly providing a ```DataCollatorForSeq2Seq``` to the Trainer. This collator correctly pads sequences within each batch to the maximum length in that batch, using the tokenizer's pad token for inputs/attention masks and -100 for labels (to ignore them in the loss calculation).

### CODE REVIEW

**Comment by rupaut98** [comment]
@darkacorn thank you!

**Comment by Etherll** [comment]
Hey, can you remove the comma after DataCollatorForSeq2Seq() it causes issues
```py
data_collator = DataCollatorForSeq2Seq(
...
), # <-------
```

**Comment by rupaut98** [comment]
@Etherll Didn't realize I had that extra "," there. Just fixed it! Thank you!

**Comment by shimmyshimmer** [comment]
Thanks guys!
